src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './Navbar.css';
+
+const Navbar = () => {
+  return (
+    <header className="navbar home-navbar">
+      <div className="logo-container">
+        <h2>
+          <a href="#">Bijutsu</a>
+        </h2>
+      </div>
+      <div className="search-container">
+        <input
+          className="textbox"
```

Comment:
Suggestion: you can make search bar as component and use it here

Reply:
Thanks for the suggestion will work on it.